src/detect-dataset.c

```diff
@@ -181,7 +184,7 @@ static int DetectDatasetParse(const char *str,
                         "'load' can only appear once");
                     return -1;
                 }
-                SCLogDebug("load %s", val);
+                SCLogDebug("load %s size: %lu", val, load_size);
```

Comment:
```
Invalid printf with size_t (not casted to uintmax_t) at /builds/inliniac/suricata-ci/suricata/src/detect-dataset.c:187
Makefile:545: recipe for target 'check' failed
```
To print a `sizet_t` use `SCLogDebug("load %s size: %"PRIuMAX, val, (uintmax_t)load_size);`